app/addons/databases/components.react.jsx

```diff
@@ -0,0 +1,321 @@
+      var graveyardTitle = "This database has just " + row.status.numDocs() +
+        " docs and " + row.status.numDeletedDocs() + " deleted docs";
+      return (
+        <i className="js-db-graveyard icon icon-exclamation-sign" ref="myself" title={graveyardTitle}></i>
+      );
+    }
+  });
+
+  var RightDatabasesHeader = React.createClass({
+
+    render: function () {
+      return (
+        <div className="header-right">
+          <AddDatabaseWidget />
+          <JumpToDatabaseWidget />
+        </div>
+      );
+    }
+  });
+
+  var AddDatabaseWidget = React.createClass({
+
+    onTrayToggle: function (e) {
+      e.preventDefault();
+      this.refs.newDbTray.toggle(function (shown) {
+        if (shown) {
+          this.refs.newDbName.getDOMNode().focus();
+        }
+      }.bind(this));
+    },
+
+    onKeyUpInInput: function (e) {
+      if (e.which === 13) {
+        this.onAddDatabase();
+      }
+    },
+
+    componentDidMount: function () {
+      databasesStore.on('change', this.onChange, this);
+    },
+
+    componentWillUnmount: function () {
+      databasesStore.off('change', this.onChange, this);
+    },
+
+    onChange: function () {
+      if (this.isMounted()) {
+        this.refs.newDbTray.setVisible(databasesStore.isPromptVisible());
+      }
+    },
+
+    onAddDatabase: function () {
+      var databaseName = this.refs.newDbName.getDOMNode().value;
+      Actions.createNewDatabase(databaseName);
+    },
+
+    render: function () {
+
+      return (
+        <div className="button" id="add-db-button">
+          <a id="add-new-database" href="#" className="add-new-database-btn" onClick={this.onTrayToggle} data-bypass="true">
+                <i className="header-icon fonticon-new-database"></i>
+                Add New Database
+          </a>
+          <ComponentsReact.Tray ref="newDbTray" className="new-database-tray">
+            <span className="add-on">Add New Database</span>
+            <input id="js-new-database-name" type="text" onKeyUp={this.onKeyUpInInput} ref="newDbName" className="input-xxlarge" placeholder="Name of database" />
+            <a className="btn" id="js-create-database" onClick={this.onAddDatabase}>Create</a>
+          </ComponentsReact.Tray>
+        </div>
+      );
+    }
+  });
+
+  var JumpToDatabaseWidget = React.createClass({
+
+    getStoreState: function () {
+      return {
+        databaseNames: databasesStore.getDatabaseNames()
+      };
+    },
+
+    getInitialState: function () {
+      return this.getStoreState();
+    },
+
+    componentDidMount: function () {
+      databasesStore.on('change', this.onChange, this);
+      $(this.refs.searchDbName.getDOMNode()).typeahead({
+        source: this.state.databaseNames,
+        updater: function (item) {
+          this.jumpToDb(item);
+        }.bind(this)
+      });
+      // turn of browser autocomplete (direct markup is not passed on)
+      $(this.refs.searchDbName.getDOMNode()).attr("autocomplete", "off");
```

Comment:
no need for the hack, as promised i searched: it is `autoComplete` (note the camelCase): https://facebook.github.io/react/docs/tags-and-attributes.html#html-attributes